spec/features/home_flow_spec.rb

```diff
@@ -0,0 +1,76 @@
+require 'spec_helper'
+
+feature 'Home Flow' do
+  let(:guide) { create(:guide) }
+  let!(:exercise) { create(:exercise, id: 1, guide: guide) }
```

Comment:
That the variable is eager instead of lazy. Let's in rspec create variables on demand